### ISSUE TITLE
[REVIEW]: FielDHub: An R Shiny Package for Design of Experiments in Life Sciences

### ISSUE THREAD

**Issue by whedon** [openjournals/joss-reviews#3122]
**Submitting author:** @DidierMurilloF (<a href="http://orcid.org/0000-0003-3685-4959">Didier Murillo Florez</a>)
**Repository:** <a href="https://github.com/DidierMurilloF/FielDHub" target ="_blank">https://github.com/DidierMurilloF/FielDHub</a>
**Version:** v0.1.0
**Editor:** @csoneson
**Reviewer:** @Prof-ThiagoOliveira, @dlebauer
**Archive:** <a href="https://doi.org/10.5281/zenodo.4768611" target="_blank">10.5281/zenodo.4768611</a>

**:warning: JOSS reduced service mode :warning:**

Due to the challenges of the COVID-19 pandemic, JOSS is currently operating in a "reduced service mode". You can read more about what that means in [our blog post](https://blog.joss.theoj.org/2020/05/reopening-joss).

## Status

[![status](https://joss.theoj.org/papers/c686579ab789fb4340baef052d3bf311/status.svg)](https://joss.theoj.org/papers/c686579ab789fb4340baef052d3bf311)

Status badge code:

```
HTML: <a href="https://joss.theoj.org/papers/c686579ab789fb4340baef052d3bf311"><img src="https://joss.theoj.org/papers/c686579ab789fb4340baef052d3bf311/status.svg"></a>
Markdown: [![status](https://joss.theoj.org/papers/c686579ab789fb4340baef052d3bf311/status.svg)](https://joss.theoj.org/papers/c686579ab789fb4340baef052d3bf311)
```
**Reviewers and authors**:

Please avoid lengthy details of difficulties in the review thread. Instead, please create a new issue in the <a href="https://github.com/DidierMurilloF/FielDHub" target="_blank">target repository</a> and link to those issues (especially acceptance-blockers) by leaving comments in the review thread below. (For completists: if the target issue tracker is also on GitHub, linking the review thread in the issue or vice versa will create corresponding breadcrumb trails in the link target.)

## Reviewer instructions & questions

@Prof-ThiagoOliveira & @dlebauer, please carry out your review in this issue by updating the checklist below. If you cannot edit the checklist please:
1. Make sure you're logged in to your GitHub account
2. Be sure to accept the invite at this URL: https://github.com/openjournals/joss-reviews/invitations

The reviewer guidelines are available here: https://joss.readthedocs.io/en/latest/reviewer_guidelines.html. Any questions/concerns please let @csoneson know.

✨ **Please start on your review when you are able, and be sure to complete your review in the next six weeks, at the very latest** ✨ 

### Conflict of interest

- [x] I confirm that I have read the [JOSS conflict of interest (COI) policy](https://github.com/openjournals/joss/blob/master/COI.md) and that: I have no COIs with reviewing this work or that any perceived COIs have been waived by JOSS for the purpose of this review.

### Code of Conduct

- [x] I confirm that I read and will adhere to the [JOSS code of conduct](https://joss.theoj.org/about#code_of_conduct).

### General checks

- [x] **Repository:** Is the source code for this software available at the <a target="_blank" href="https://github.com/DidierMurilloF/FielDHub">repository url</a>?
- [x] **License:** Does the repository contain a plain-text LICENSE file with the contents of an [OSI approved](https://opensource.org/licenses/alphabetical) software license?
- [x] **Contribution and authorship:** Has the submitting author (@DidierMurilloF) made major contributions to the software? Does the full list of paper authors seem appropriate and complete?
- [x] **Substantial scholarly effort:** Does this submission meet the scope eligibility described in the [JOSS guidelines](https://joss.readthedocs.io/en/latest/submitting.html#substantial-scholarly-effort)

### Functionality

- [x] **Installation:** Does installation proceed as outlined in the documentation?
- [x] **Functionality:** Have the functional claims of the software been confirmed?
- [x] **Performance:** If there are any performance claims of the software, have they been confirmed? (If there are no claims, please check off this item.)

### Documentation

- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
- [x] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).
- [x] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?
- [x] **Automated tests:** Are there automated tests or manual steps described so that the functionality of the software can be verified?
- [x] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support

### Software paper

- [x] **Summary:** Has a clear description of the high-level functionality and purpose of the software for a diverse, non-specialist audience been provided?
- [x] **A statement of need:** Does the paper have a section titled 'Statement of Need' that clearly states what problems the software is designed to solve and who the target audience is?
- [x] **State of the field:** Do the authors describe how this software compares to other commonly-used packages?
- [x] **Quality of writing:** Is the paper well written (i.e., it does not require editing for structure, language, or writing quality)?
- [x] **References:** Is the list of references complete, and is everything cited appropriately that should be cited (e.g., papers, datasets, software)? Do references in the text use the proper [citation syntax]( https://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html#citation_syntax)?

### Conflict of interest

- [x] I confirm that I have read the [JOSS conflict of interest (COI) policy](https://github.com/openjournals/joss/blob/master/COI.md) and that: I have no COIs with reviewing this work or that any perceived COIs have been waived by JOSS for the purpose of this review.

### Code of Conduct

- [x] I confirm that I read and will adhere to the [JOSS code of conduct](https://joss.theoj.org/about#code_of_conduct).

### General checks

- [x] **Repository:** Is the source code for this software available at the <a target="_blank" href="https://github.com/DidierMurilloF/FielDHub">repository url</a>?
- [x] **License:** Does the repository contain a plain-text LICENSE file with the contents of an [OSI approved](https://opensource.org/licenses/alphabetical) software license?
- [x] **Contribution and authorship:** Has the submitting author (@DidierMurilloF) made major contributions to the software? Does the full list of paper authors seem appropriate and complete?
- [x] **Substantial scholarly effort:** Does this submission meet the scope eligibility described in the [JOSS guidelines](https://joss.readthedocs.io/en/latest/submitting.html#substantial-scholarly-effort)

### Functionality

- [x] **Installation:** Does installation proceed as outlined in the documentation?
- [x] **Functionality:** Have the functional claims of the software been confirmed?
- [x] **Performance:** If there are any performance claims of the software, have they been confirmed? (If there are no claims, please check off this item.)

### Documentation

- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
- [x] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).
- [x] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?
- [x] **Automated tests:** Are there automated tests or manual steps described so that the functionality of the software can be verified?
- [x] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support

### Software paper

- [x] **Summary:** Has a clear description of the high-level functionality and purpose of the software for a diverse, non-specialist audience been provided?
- [x] **A statement of need:** Does the paper have a section titled 'Statement of Need' that clearly states what problems the software is designed to solve and who the target audience is?
- [x] **State of the field:** Do the authors describe how this software compares to other commonly-used packages?
- [x] **Quality of writing:** Is the paper well written (i.e., it does not require editing for structure, language, or writing quality)?
- [x] **References:** Is the list of references complete, and is everything cited appropriately that should be cited (e.g., papers, datasets, software)? Do references in the text use the proper [citation syntax]( https://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html#citation_syntax)?

**Comment by whedon** [comment]
Hello human, I'm @whedon, a robot that can help you with some common editorial tasks. @Prof-ThiagoOliveira, @dlebauer it looks like you're currently assigned to review this paper :tada:.

**:warning: JOSS reduced service mode :warning:**

Due to the challenges of the COVID-19 pandemic, JOSS is currently operating in a "reduced service mode". You can read more about what that means in [our blog post](https://blog.joss.theoj.org/2020/05/reopening-joss).

**:star: Important :star:**

If you haven't already, you should seriously consider unsubscribing from GitHub notifications for this (https://github.com/openjournals/joss-reviews) repository. As a reviewer, you're probably currently watching this repository which means for GitHub's default behaviour you will receive notifications (emails) for all reviews 😿

To fix this do the following two things:

1. Set yourself as 'Not watching' https://github.com/openjournals/joss-reviews:

![watching](https://cloud.githubusercontent.com/assets/4483/20250593/64d7ce48-a9de-11e6-9225-d3dfb3e48e68.png)

2. You may also like to change your default settings for this watching repositories in your GitHub profile here: https://github.com/settings/notifications

![notifications](https://cloud.githubusercontent.com/assets/4483/20250604/79eb3144-a9de-11e6-9e51-0fe940966689.png)


For a list of things I can do to help you, just type:

```
@whedon commands
```

For example, to regenerate the paper pdf after making changes in the paper's md or bib files, type:

```
@whedon generate pdf
```

**Comment by whedon** [comment]
```
Software report (experimental):

github.com/AlDanial/cloc v 1.88  T=0.27 s (490.4 files/s, 102676.0 lines/s)
-------------------------------------------------------------------------------
Language                     files          blank        comment           code
-------------------------------------------------------------------------------
HTML                            30           1365            357          10628
R                               81           1774           1785           9884
CSS                              5            148             70            642
JavaScript                       3             64             32            256
Markdown                         5             72              0            182
TeX                              1              5              0             51
YAML                             4              4              3             43
SVG                              1              0              1             11
Rmd                              1             12             31              6
-------------------------------------------------------------------------------
SUM:                           131           3444           2279          21703
-------------------------------------------------------------------------------


Statistical information for the repository '0760439efd04e1b7cebeea64' was
gathered on 2021/03/18.
The following historical commit information, by author, was found:

Author                     Commits    Insertions      Deletions    % of changes
DidierMurillo                    1           352              0          100.00

Below are the number of rows from each author that have survived and are still
intact in the current revision:

Author                     Rows      Stability          Age       % in comments
DidierMurillo               352          100.0          0.0                9.09
```

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.1111/j.1467-842X.2011.00642.x is OK
- 10.1198/108571106X154443 is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.03122/joss.03122/10.21105.joss.03122.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.03122/joss.03122/10.21105.joss.03122.pdf) :page_facing_up: :point_left:

**Comment by csoneson** [comment]
👋 @DidierMurilloF, @Prof-ThiagoOliveira, @dlebauer - this is the review thread for the submission. All of our communications will happen here from now on.

Both reviewers have checklists at the top of this thread. As you go over the submission, please check any items that you feel have been satisfied. There are also links to the JOSS reviewer guidelines.

The JOSS review is different from most other journals. Our goal is to work with the authors to help them meet our criteria instead of merely passing judgment on the submission. As such, the reviewers are encouraged to submit issues directly in the software repository. If you do so, please mention this thread so that a link is created (and I can keep an eye on what is happening). Please also feel free to comment and ask questions in this thread. It is often easier to post comments/questions/suggestions as you come across them instead of waiting until you've reviewed the entire package.

Please feel free to ping me (@csoneson) if you have any questions or concerns. Thanks!

**Comment by dlebauer** [comment]
PR submitted w/ OSI approved license https://github.com/DidierMurilloF/FielDHub/pull/1

**Comment by Prof-ThiagoOliveira** [comment]
I've been submitting issues with partial solutions to #3107

**Comment by Prof-ThiagoOliveira** [comment]
PR submitted with changes [DidierMurilloF/FielDHub#5](https://github.com/DidierMurilloF/FielDHub/pull/5)

**Comment by Prof-ThiagoOliveira** [comment]
PR submitted with changes on fct_full_factorial.R. [DidierMurilloF/FielDHub#6](https://github.com/DidierMurilloF/FielDHub/pull/6)

**Comment by Prof-ThiagoOliveira** [comment]
Issue 7 - [DidierMurilloF/FielDHub#7](https://github.com/DidierMurilloF/FielDHub/issues/7)

**Comment by Prof-ThiagoOliveira** [comment]
Issue 8 - [DidierMurilloF/FielDHub#8](https://github.com/DidierMurilloF/FielDHub/issues/8)

**Comment by Prof-ThiagoOliveira** [comment]
Issue 8 - [DidierMurilloF/FielDHub#8](https://github.com/DidierMurilloF/FielDHub/issues/9)

**Comment by Prof-ThiagoOliveira** [comment]
Issue 10 - [DidierMurilloF/FielDHub#10](https://github.com/DidierMurilloF/FielDHub/issues/10)

**Comment by Prof-ThiagoOliveira** [comment]
Issue 11 - [DidierMurilloF/FielDHub#11](https://github.com/DidierMurilloF/FielDHub/issues/11)

**Comment by csoneson** [comment]
Hi all - just wanted to check in to see how things are going. I see that @Prof-ThiagoOliveira has submitted a number of issues that are being actively worked on. @dlebauer - could you give us a quick update on where things are on your side? Just ping me if there are any questions. Thanks!

**Comment by Prof-ThiagoOliveira** [comment]
@csoneson I am waiting for the authors to update my suggestions. Where could I include suggestions about the manuscript?

**Comment by csoneson** [comment]
Thanks @Prof-ThiagoOliveira! You can add comments on theh paper either as posts in this thread or open an issue in the software repository as you have done above.

**Comment by csoneson** [comment]
@dlebauer - could you update us on how your review is going? Thanks!

**Comment by dlebauer** [comment]
@csoneson this is my first review for JOSS. Just learning how this works. One of the first things I usually do when reviewing an article is see if it meets journal requirements. In this case I started by submitting a pr to meet the journal requirements of having an OSI approved license. However it hasn't been merged or addressed (eg with a different license). While this doesn't prevent a full review, I was hoping this would be a first step and would establish the pattern for using issues and prs as a mechanism for review as suggested by JOSS, and which I think is a great idea.

**Comment by csoneson** [comment]
Thanks @dlebauer. From the DESCRIPTION file it looks like the package is indeed MIT licensed. @DidierMurilloF - could you clarify the situation here (and add the license to the repository as well)?

**Comment by dlebauer** [comment]
Now I see it looks like the issue is that there are two files ... LICENSE and LICENSE.md, and github is only picking up the LICENSE. I will update the PR and proceed

**Comment by dlebauer** [comment]
- https://github.com/DidierMurilloF/FielDHub/issues/14
- https://github.com/DidierMurilloF/FielDHub/issues/13
- https://github.com/DidierMurilloF/FielDHub/issues/12

**Comment by csoneson** [comment]
👋🏻 Hi all, I just wanted to check in and see that everything is on track and moving forward here. Let me know if there are any questions or roadblocks. Thanks!

**Comment by dlebauer** [comment]
@csoneson still waiting on author to respond to issues / PRs

**Comment by csoneson** [comment]
@DidierMurilloF - could you give us an update on where things are on your side? Let us know if you have any questions. Thanks!

**Comment by DidierMurilloF** [comment]
@csoneson We are working on the last requests from the editors. We think that is going to be ready this week. Thank you! 

Didier Murillo.

**Comment by csoneson** [comment]
Great, thanks @DidierMurilloF! No stress, just wanted to check in and make sure that there were no major roadblocks or questions.

**Comment by csoneson** [comment]
👋🏻 @DidierMurilloF - just another quick check-in to make sure things are on track. I can see that there have been changes to the repository in the past few days to address the reviewers' comments, so I assume everything is under control; just wanted to double-check. Thanks!

**Comment by DidierMurilloF** [comment]
Dear @csoneson,

Thank you very much for following up on the progress of our App. We have addressed all the suggestions that reviewers have made for the first release of FielDHub and we are committed to doing all the improvements suggested by reviewers for the next version. We are very pleased with all the improvements that have been suggested by the reviewers @Prof-ThiagoOliveira, @dlebauer and we believe this has made our application stronger and better.  We are looking forward to the acceptance of our App at the JOSS journal. Again, thank you very much and we look forward to hearing from you soon. 

Best,

Didier Murillo.

**Comment by dlebauer** [comment]
Really nice work @DidierMurilloF ! I've checked off all of the reviewer rubric. Now it is just waiting on @Prof-ThiagoOliveira 's review.

**Comment by csoneson** [comment]
Perfect - thanks!

**Comment by Prof-ThiagoOliveira** [comment]
I agree with @dlebauer. You've done a great job and addressed all our comments. I've checked the package, and it is ready to be accepted as a publication on the JOSS. I am happy to see you have gone beyond our comments. Please, @csoneson, consider the paper accepted by my side.

**Comment by csoneson** [comment]
Thanks @Prof-ThiagoOliveira! 

@DidierMurilloF - I will have a quick look through as well, and get back to you shortly with the next steps.

**Comment by csoneson** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.03122/joss.03122/10.21105.joss.03122.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.03122/joss.03122/10.21105.joss.03122.pdf) :page_facing_up: :point_left:

**Comment by csoneson** [comment]
@whedon check references

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.1111/j.1467-842X.2011.00642.x is OK
- 10.1198/108571106X154443 is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by csoneson** [comment]
@DidierMurilloF - this looks good! The only thing I would suggest is on line 27 in the paper, to change ``` it allows users to specify the dimensions of the field as row and columns``` to ``` it allows users to specify the dimensions of the field (the number of rows and columns)``` or something along those lines. 

The next steps would then be as follows: 
- [ ] Make a tagged release of the software, and list the version tag of the archived version here.
- [ ] Archive the release in Zenodo or a similar service
- [ ] Check that the archival deposit (e.g., in Zenodo) has the correct metadata. In particular, the title and author list should match those of the paper. You may also add the authors' ORCID.
- [ ] Please list the DOI of the archived version here.

I can then move forward with accepting the submission.

**Comment by DidierMurilloF** [comment]
Dear @csoneson 

I did the change you suggested in the paper, thank you. I am not familiar with the steps above, but I will learn about them as soon as possible. 

Could you please indicate how should we tag the release of the software? 

Best,

Didier Murillo

**Comment by csoneson** [comment]
Sure! Here's how you create a release: https://docs.github.com/en/github/administering-a-repository/managing-releases-in-a-repository. We prefer that you tag it with the version of the package (e.g., `0.1.0`). 

And here's a guide to archiving the repository in Zenodo: https://guides.github.com/activities/citable-code/

**Comment by csoneson** [comment]
Please also generate a new pdf with `@whedon generate pdf` once the indicated change to the paper has been pushed to the GitHub repository, so that we have the latest version.

**Comment by DidierMurilloF** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.03122/joss.03122/10.21105.joss.03122.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.03122/joss.03122/10.21105.joss.03122.pdf) :page_facing_up: :point_left:

**Comment by DidierMurilloF** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.03122/joss.03122/10.21105.joss.03122.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.03122/joss.03122/10.21105.joss.03122.pdf) :page_facing_up: :point_left:

**Comment by DidierMurilloF** [comment]
Dear @csoneson,

Thank you for your help. I got all steps listed above. Now `FielDHub` is available at Zenodo. 

The corresponding DIO is the following,

http://doi.org/10.5281/zenodo.4768155

I also did small changes to the title of the paper to match with the release at R CRAN. Thank you!

Best,

Didier Murillo.

**Comment by csoneson** [comment]
@whedon set 10.5281/zenodo.4768611 as archive

**Comment by whedon** [comment]
OK. <a href="https://doi.org/10.5281/zenodo.4768611" target="_blank">10.5281/zenodo.4768611</a> is the archive.

**Comment by csoneson** [comment]
@whedon set v0.1.0 as version

**Comment by whedon** [comment]
OK. v0.1.0 is the version.

**Comment by csoneson** [comment]
@whedon accept

**Comment by whedon** [comment]
```
Attempting dry run of processing paper acceptance...
```

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.1111/j.1467-842X.2011.00642.x is OK
- 10.1198/108571106X154443 is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by whedon** [comment]
:wave: @openjournals/joss-eics, this paper is ready to be accepted and published.

 Check final proof :point_right: https://github.com/openjournals/joss-papers/pull/2316

If the paper PDF and Crossref deposit XML look good in https://github.com/openjournals/joss-papers/pull/2316, then you can now move forward with accepting the submission by compiling again with the flag `deposit=true` e.g.
 ```
@whedon accept deposit=true
```

**Comment by csoneson** [comment]
Thanks @DidierMurilloF! I'm handing over now to the Associate Editor in Chief on rotation for the final steps.

**Comment by DidierMurilloF** [comment]
Dear @csoneson thank you very much for all your help. We look forward to hearing more about the next steps. 

Best regards,

Didier Murillo

**Comment by kyleniemeyer** [comment]
Hi @DidierMurilloF, I'm the AEIC on duty this week doing some final checks before publishing. I noticed some minor issues in the references, could you merge this PR I made to fix them? https://github.com/DidierMurilloF/FielDHub/pull/16

**Comment by DidierMurilloF** [comment]
Dear @kyleniemeyer thank you for your help! I merged the pull request.  Please, let me know if you need anything else from me.

Best,

Didier Murillo.

**Comment by kyleniemeyer** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.03122/joss.03122/10.21105.joss.03122.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.03122/joss.03122/10.21105.joss.03122.pdf) :page_facing_up: :point_left:

**Comment by kyleniemeyer** [comment]
@whedon accept deposit=true

**Comment by whedon** [comment]
```
Doing it live! Attempting automated processing of paper acceptance...
```

**Comment by whedon** [comment]
🐦🐦🐦 👉 [Tweet for this paper](https://twitter.com/JOSS_TheOJ/status/1394726160790081541) 👈 🐦🐦🐦

**Comment by whedon** [comment]
🚨🚨🚨 **THIS IS NOT A DRILL, YOU HAVE JUST ACCEPTED A PAPER INTO JOSS!** 🚨🚨🚨

 Here's what you must now do:

0. Check final PDF and Crossref metadata that was deposited :point_right: https://github.com/openjournals/joss-papers/pull/2320
1. Wait a couple of minutes, then verify that the paper DOI resolves [https://doi.org/10.21105/joss.03122](https://doi.org/10.21105/joss.03122)
2. If everything looks good, then close this review issue.
3. Party like you just published a paper! 🎉🌈🦄💃👻🤘

 Any issues? Notify your editorial technical team...

**Comment by kyleniemeyer** [comment]
Congrats @DidierMurilloF on your article's publication in JOSS!

Many thanks to @Prof-ThiagoOliveira and @dlebauer for reviewing this, and @csoneson for editing.

**Comment by whedon** [comment]
:tada::tada::tada: Congratulations on your paper acceptance! :tada::tada::tada:

If you would like to include a link to your paper from your README use the following code snippets:

```
Markdown:
[![DOI](https://joss.theoj.org/papers/10.21105/joss.03122/status.svg)](https://doi.org/10.21105/joss.03122)

HTML:
<a style="border-width:0" href="https://doi.org/10.21105/joss.03122">
  <img src="https://joss.theoj.org/papers/10.21105/joss.03122/status.svg" alt="DOI badge" >
</a>

reStructuredText:
.. image:: https://joss.theoj.org/papers/10.21105/joss.03122/status.svg
   :target: https://doi.org/10.21105/joss.03122
```
This is how it will look in your documentation:

[![DOI](https://joss.theoj.org/papers/10.21105/joss.03122/status.svg)](https://doi.org/10.21105/joss.03122)

**We need your help!**

Journal of Open Source Software is a community-run journal and relies upon volunteer effort. If you'd like to support us please consider doing either one (or both) of the the following:

- Volunteering to review for us sometime in the future. You can add your name to the reviewer list here: https://joss.theoj.org/reviewer-signup.html
- Making a small donation to support our running costs here: https://numfocus.org/donate-to-joss
